### ISSUE TITLE
Search broken

### ISSUE THREAD

**Issue by jowi24** [nextcloud/tasks#22]
### Steps to reproduce
1. Open task app
2. Click on the magnifier icon in the upper right of the page (or press Ctrl+F)
3. Enter a word that is part of an existing task
4. Press enter

### Expected behaviour
A filtered list of tasks that contain the word should appear

### Actual behaviour
Nothing happens.

### Server configuration
**Operating system**:
Arch Linux
**Web server:**
Nginx
**Database:**
Mysql
**PHP version:**
7.0
**ownCloud version:** (see ownCloud admin page)
10.0.1
**Tasks version:** (see ownCloud apps page)
0.9.3
**Updated from an older ownCloud or fresh install:**
Update from 10.0
**Signing status (ownCloud 9.0 and above):**

```
No errors have been found.
```

**List of activated apps:**

```
Enabled:
  - activity: 2.3.2
  - admin_audit: 1.0.0
  - bookmarks: 0.9.0
  - calendar: true
  - comments: 1.0.0
  - contacts: 1.5.1
  - dav: 1.0.1
  - external: true
  - federatedfilesharing: 1.0.1
  - federation: 1.0.1
  - files: 1.5.2
  - files_external: 1.0.2
  - files_pdfviewer: 0.8.1
  - files_sharing: 1.0.0
  - files_texteditor: 2.1
  - files_trashbin: 1.0.0
  - files_versions: 1.3.0
  - files_videoplayer: 0.9.8
  - firstrunwizard: 1.1
  - gallery: 15.0.0
  - notifications: 0.3.0
  - password_policy: 1.0.0
  - provisioning_api: 1.0.0
  - serverinfo: 1.1.1
  - survey_client: 0.1.5
  - systemtags: 1.0.2
  - tasks: true
  - templateeditor: 0.1
  - theming: 1.0.1
  - updatenotification: 1.0.1
  - workflowengine: 1.0.1
Disabled:
  - encryption
  - files_accesscontrol
  - files_automatedtagging
  - files_retention
  - twofactor_totp
  - user_external
  - user_ldap
  - user_saml
```

**The content of config/config.php:**

```

    "system": {
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "datadirectory": "\/srv\/http\/data\/",
        "dbtype": "mysql",
        "version": "9.1.1.5",
        "dbname": "nextcloud",
        "dbhost": "localhost",
        "dbtableprefix": "oc_",
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "logtimezone": "UTC",
        "installed": true,
        "logfile": "\/var\/log\/nginx\/nextcloud.log",
        "loglevel": 2,
        "theme": "",
        "maintenance": false,
        "skeletondirectory": "",
        "appstore.experimental.enabled": true
    }
}
```

**Are you using external storage, if yes which one:** local/smb/sftp/...
no
**Are you using encryption:** yes/no
no
**Are you using an external user-backend, if yes which one:** LDAP/ActiveDirectory/Webdav/...
no
### Client configuration
**Browser:**
Chrome
**Operating system:**
Windows 7
**CalDAV-clients:**
n/a

**Comment by avanc** [comment]
I can confirm the behavior on my installation.

tasks version: 0.9.4

**Comment by raimund-schluessler** [comment]
Search is broken atm.

**Comment by ferdiga** [comment]
still broken in nextcloud 12.0.5, task 0.9.6

**Comment by BarthyB** [comment]
"Nothing Happens" is not quite true (anymore?).

I can see a container below my task list with a loading icon/spinner and a "Searching other places" label.
Also, a file called `searchprovider.js` logs "scrolling to 840" hundreds of times.

Tasks version 0.9.7
Nextcloud version 14.0.1

**Comment by raimund-schluessler** [comment]
Fix is in #216
